src/EventListener/AddToSearchIndexListener.php

```diff
@@ -27,7 +41,7 @@ class AddToSearchIndexListener
      */
     public function onKernelTerminate(PostResponseEvent $event)
     {
-        if (!defined('TL_ROOT')) {
+        if ($this->framework->isInitialized()) {
```

Comment:
Could this be right? Shouldn´t is be `if (!$this->framework->isInitialized()) {` ???